packages/globals/src/base-component/window-manager.ts

```diff
@@ -0,0 +1,62 @@
+export type ScrollState = { scrollY: number; width: number; height: number };
```

Comment:
questo file non lo metterei sotto la cartella base-component, ma direttamente sotto src o in una sua cartella dedicata